deploy_local_testnet.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -i
+
```

Comment:
I suggest to add `set -euo pipefail` to make the script behavior more predictable.